jaeger_client/span_context.py

```diff
@@ -15,14 +15,22 @@
 from __future__ import absolute_import
 
 import opentracing
+from typing import Optional
 
 
 class SpanContext(opentracing.SpanContext):
     __slots__ = ['trace_id', 'span_id', 'parent_id', 'flags',
                  '_baggage', '_debug_id']
 
     """Implements opentracing.SpanContext"""
-    def __init__(self, trace_id, span_id, parent_id, flags, baggage=None, debug_id=None):
+    def __init__(
+        self, trace_id: Optional[int],
+        span_id: Optional[int],
+        parent_id: Optional[int],
+        flags: Optional[int],
```

Comment:
Hm, is there any code that actually creates context with None for these fields? We treat 0 as invalid value, but otherwise IDs are ints.

Reply:
Yes, they can get None: SpanContext.with_debug_id()

Reply:
that function is deprecated, we can change it to initialize with 0s instead of None's if that's the only reason.

Reply:
I reset the files anyway. Can be done in a separate commit (you might want to create an issue?)